setup.py

```diff
@@ -25,20 +29,17 @@
     long_description = f.read()
 
 setup(name='sqlalchemy_drill',
-      version='0.3.0',
+      version='0.3.1',
```

Comment:
What would you think about calling this version 1.0?

Reply:
Yes good call, too many projects sit in denial in 0.x for years.  I will push this in a couple of minutes.